src/Pokemon.java

```diff
@@ -0,0 +1,74 @@
+import java.util.List;
+
+public abstract class Pokemon {
+    private final String name;
+    private final int level;
+    private int hp;
+    private final String food;
+    private final String sound;
+    private List<String> attacks;
+    private final String type;
+    private double power = 1;
+    protected double damageMultiplier;
+
+    public Pokemon(String name, int level, int hp, String food, String sound, String type) {
```

Comment:
Je zet niet al je klasse variabelen in de constructor. Bijvoorbeeld attacks, maar het is natuurlijk niet de bedoeling dat je een Pokemon maakt zonder attacks.